website/docs/events/m3-slides/.slides/introduction.md

```diff
@@ -1,8 +1,100 @@
 ---
 
-## Introduction
+## Open Education Hub
+
+----
+
+![Moodle Block](media/screenshot-os-moodle-block.png)
+
+----
+
+![Moodle Results](media/screenshot-os-moodle-results.png)
+
+----
+
+![GitLab Repositories](media/screenshot-os-gitlab-list.png)
+
+----
+
+![GitLab Submissions](media/screenshot-os-gitlab-submissions.png)
+
+----
+
+![Student Issues](media/screenshot-os-student-submission.png)
+
+----
+
+![Student Contributions](media/screenshot-os-student-contrib.png)
+
+----
+
+![Automate Everything](media/automate-everything.jpg)
+
+----
+
+![Simplify Everything](media/simplify-everything.jpg)
+
+----
+
+![Reuse Everything](media/reuse-everything.jpg)
+
+----
+
+![Document Everything](media/document-everything.jpg)
+
+----
+
+![Publicize Everything](media/publicize-everything.jpg)
+
+----
+
+![Involve Everyone](media/involve-everyone.jpg)
```

Comment:
Wouldn't it make more sense to reverse the order of these images? I think it's better to start with the intentions ("* Everyone/Everything") and then follow up with examples.